### DISCUSSION TITLE
[NFT] New releases unreachable

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2025] [General]
How to get 10 golds + 500 wood to try to mint Wood Nymph Wendy in 12 days? This a good boost for who dont have any.

Cyborg Bear is decoration so maybe it can be unreachable for everyone.

The Wood Nymph Wendy becomes more necessary for those who have T3 NFT or those who don't have any boost?

I m playing since beta, it is not unreachable to me, imagine who started with LE.

I do not believe that this division is healthy and attractive to new players.

Same happens to Goblin Craft, unreachable for everyone.

If you don't invest, you will always feel late. Who is starting the game now must see an abyss, not attractive.

**Comment by Aminbtdds** [comment]
Imho, my guess is devs want to slow down or reverse gold total supply growth,  while we have iron problem(in terms of node count and 8hr cooldown) and woods and stones deflating total supply, which will result in 20-30% margin for gold production just like beta.
Not everybody can mint every nfts in game, which is logical imo, people should spend to have them all, both time and money, we are shifting from f2p or better said low cost p2p with slogan of play to own, to something like pay to play in different levels.
New players who want to own anything valuable, they should pay for it(like 2 or 3 weeks of slf earnings to buy gold).
We will be different game down this road, which will need different kind of marketing and everything else.
We are 5k players, we need people to join to growth, new people are not beta players who love and play sfl for more than a year like us.
Hope everything will work well in the terms of game appeal🌻❤
